components/ExerciseApp.js

```diff
@@ -30,4 +36,22 @@ const styles = StyleSheet.create({
   },
 });
 
-export default ExerciseApp;
+export default firebaseConnect([
```

Comment:
This is a little confusing.  The 'connect' method works jsut the same as usual.

The firebaseConnect essentially populates the {
    firebase: {
        data: ...
    }
}
key inside of the redux store, with whatever key you requested.  In this case, it's "test".

Also, there are 2 levels of higher order component nesting